tls/s2n_security_policies.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include "tls/s2n_cipher_preferences.h"
+#include "tls/s2n_kem_preferences.h"
+
+struct s2n_security_policy {
+    uint8_t minimum_protocol_version;
+    const struct s2n_cipher_preferences *cipher_preferences;
+    const struct s2n_kem_preferences *kem_preferences;
+};
+
+extern const struct s2n_security_policy security_policy_20140601;
```

Comment:
since these are new, good chance to get rid of externs?

Reply:
`extern` is actually required here to access these variables